core/src/main/java/io/ap4k/Resources.java

```diff
@@ -90,6 +88,15 @@ public void add(String group, HasMetadata metadata) {
     groups.get(group).addToItems(metadata);
   }
 
+  /**
+   * Add a {@link Visitor} to the specified resource explicit group.
+   * Explicit groups are groups that only get resources and decorators that have been explicitly specified.
```

Comment:
What will be then happen if they are `explicitly` declared ? This code source doc should be improved